BuildResidentialHPXML/measure.rb

```diff
@@ -1190,6 +1208,18 @@ def arguments(model)
     arg.setDescription("Whether the heat pump is ducted or not. Only used for #{HPXML::HVACTypeHeatPumpMiniSplit}.")
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heat_pump_airflow_defect_ratio', false)
+    arg.setDisplayName('Cooling System: Airflow Defect Ratio')
+    arg.setDescription('The airflow defect ratio, defined as (InstalledAirflow - DesignAirflow) / DesignAirflow, of the heat pump. A value of zero means no airflow defect.')
```

Comment:
Only applies to air source, ground source, and _ducted_ mini-split. (It does not apply to a ductless mini-split.)